common-lib/src/cmr/common/api/web_server.clj

```diff
@@ -1,6 +1,7 @@
 (ns cmr.common.api.web-server
   "Defines a web server component."
   (:require
+   [clojure.core.reducers :as reducers]
```

Comment:
Woo-hoo! 🏆 and 🍺 for @chris-durbin!

(I feel shame for not having said this the first time around ... I'm going to my room now)